packages/fx-core/src/plugins/resource/sql/plugin.ts

```diff
@@ -249,9 +250,18 @@ export class SqlPluginImpl {
     managementClient: ManagementClient
   ): Promise<void> {
     let retryCount = 0;
+    const databases: { [key: string]: boolean } = {};
```

Comment:
better naming? my understading is this indicates whether add user succeedded for each database?

Reply:
Thanks.
It is fixed.